src/foundry/client/data/documents/journal-entry.d.mts

```diff
@@ -48,20 +52,12 @@ declare global {
      */
     panToNote(options?: PanToNoteOptions): Promise<void>;
 
-    protected _preCreate(
-      data: BaseJournalEntry.ConstructorData,
-      options: DocumentModificationOptions,
-      user: BaseUser,
-    ): Promise<boolean | void>;
-
-    protected _preUpdate(
-      changed: SchemaField.AssignmentType<BaseJournalEntry.Schema, {}>,
-      options: DocumentModificationOptions,
-      user: BaseUser,
-    ): Promise<boolean | void>;
+    /* -------------------------------------------- */
+    /*  Event Handlers                              */
+    /* -------------------------------------------- */
 
     protected override _onUpdate(
-      data: foundry.documents.BaseJournalEntry.UpdateData,
+      changed: fields.SchemaField.InnerAssignmentType<BaseJournalEntry.Schema>,
```

Comment:
UpdateData *is* this InnerAssignmentType, `type UpdateData = fields.SchemaField.InnerAssignmentType<Schema>;` - UpdateData is going to be more semantic and also minimizes imports.

Reply:
gotcha